framework/docker/dataavailability/node_config.go

```diff
@@ -20,20 +20,12 @@ type NodeConfig struct {
 	Env []string
 	// ConfigModifications specifies modifications to be applied to config files
 	ConfigModifications map[string]toml.Toml
-	// CustomPorts allows overriding default port configuration
-	CustomPorts *PortConfig
+	// InternalPorts allows overriding default port configuration
+	InternalPorts types.Ports
 	// postInit functions are executed sequentially after the node is initialized
 	postInit []func(ctx context.Context, node *Node) error
 }
 
-// PortConfig allows customization of node ports
-type PortConfig struct {
```

Comment:
removed this type since all the information is present in the `types.Ports` type used in the new NetworkInfo